src/python/pants/option/arg_splitter.py

```diff
@@ -206,12 +206,14 @@ def assign_flag_to_scope(flg: str, default_scope: str) -> None:
 
   @staticmethod
   def spec(arg: str) -> bool:
-    """Does arg 'look like' a Spec (rather than a goal name).
+    """Does arg 'look like' a spec (rather than a goal name).
```

Comment:
```suggestion
    """Return whether `arg` 'looks like' a spec (rather than a goal name).
```

Reply:
I also might consider renaming this method to `is_probably_a_spec()` or something, or `is_probably_not_a_goal()`, since what it's checking is unclear right now, and I wouldn't want someone to assume that something is *definitely* intended to be a spec and not a goal just because this method returned True. Since this is a publicly exported method, this level of care seems reasonable.